stylesheets/_component.navigation.scss

```diff
@@ -239,13 +239,14 @@ $nav-color-dark:  darken(color(jadu-blue, dark), 5%) !default;
     .nav-link {
         color: lighten($nav-color-light, 40);
         display: block;
-        height: 72px;
+        height: 100%;
         font-family: $font-family-regular;
         font-size: .75em;
         line-height: 1em;
         max-width: $nav-primary-width;
+        min-height: 72px;
```

Comment:
Do we think it'd be worth adding a "magic number" comment here? This would help the next person to arrive at this block understand how you got to the value 72px. https://sass-guidelin.es/#magic-numbers